README.md

```diff
@@ -137,6 +137,33 @@ data:
       - 172.18.0.100-172.18.0.255
 ```
 
+### Create a BGP Peer object
```

Comment:
Should you also add an example for address-pool using the bgp protocol?

Reply:
sure I added sample config that combine both CR